addons/nightvision/functions/fnc_blending.sqf

```diff
@@ -23,25 +23,19 @@
 
 if (!hasInterface) exitWith {};
 
-private ["_vehicle", "_weapon", "_ammo", "_magazine", "_player"];
+params ["_vehicle", "_weapon", "", "", "_ammo", "_magazine"];
 
-_vehicle = _this select 0;
-_weapon = _this select 1;
-_ammo = _this select 4;
-_magazine = _this select 5;
-
-_player = ACE_player;
```

Comment:
This is actually quicker by the way, due to the whole global variable indexing stuff. Then again, meh :wink: